buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -109,6 +109,82 @@
         "failOnStandardError": "false"
       }
     },
+    {
```

Comment:
SCEP ops was generalized and we use a script on top of all "extension tasks". This file already has these two (first and last tasks) so you don't need to add any SCEP tasks anymore

Reply:
@jcagme So the changes to this BuildTests (aside from the queues) are uneccessary?  If so, I saw some new access denied errors last night.

Reply:
Correct. There access denied errors are related to the agent not recognizing the defined user capability, hence SCEP is not being disable.